### DISCUSSION TITLE
Clearing a select combobox value

### DISCUSSION THREAD

**Discussion by richbachman** [ariakit/ariakit#1376] [Questions] [answered]
I'm looking into some of your examples. Specifically the [SelectCombobox](https://ariakit.org/examples/select-combobox). Is there a method to clear the value if one is already chosen?

I see there is a similar approach on the [Combobox cancel](https://ariakit.org/examples/combobox-cancel) example.

**Comment by diegohaz** [comment, marked as answer]
You're free to call `select.setValue("")` if you want and render whatever you want as the select children:
```jsx
<Select state={select}>
  {select.value || "Choose an item"}
  <SelectArrow />
</Select>
```

But I'd just render this as the first item instead (for the select-combobox example):

```jsx
<ComboboxItem focusOnHover className="select-item">
  {(props) => <SelectItem {...props} value="Choose an item" />}
</ComboboxItem>
```

**Reply by richbachman** [reply]
Sorry for the late reply. This helped solve my issue. Thanks for the help!
